### PR TITLE
mofo-ui initial proof of concept - modularizing RadioFilter

### PR DESCRIPTION
**Pull request by gvn**

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -1,6 +1,12 @@
 var webpack = require(`webpack`);
 
 module.exports = {
+  resolve: {
```
Comment (alanmoo) [inline webpack.config.js]: There's got to be a way to not require this, I think it'll be using devDependencies. Look at how react-router and other react dependent libraries work.

File: app/scss/main.scss [hunk 2]
```diff
@@ -80,6 +80,7 @@ h6 {
 
 // Components
 
+@import '../../node_modules/mofo-ui/dist/mofo-ui';
```
Comment (alanmoo) [inline app/scss/main.scss]: Once on npm, we should probably just be able to import the components we use in a project, as well as importing the whole thing at once.

File: webpack.config.js (hunk 1 above)
Comment (gvn) [inline webpack.config.js]: This is literally pulled from `react-router`. 

https://github.com/reactjs/react-router/blob/master/webpack.config.js#L10

File: app/scss/main.scss (hunk 2 above)
Comment (gvn) [inline app/scss/main.scss]: You actually can already if you do:

`@import '../../node_modules/mofo-ui/src/components/radio-filter/radio-filter';`

**Comment by gvn** [comment]
Ok, @alanmoo this is now a published npm module and ready for final review...

**Pull request merged**